wagtailio/navigation/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class NavigationConfig(AppConfig):
+    name = "wagtailio.navigation"
+    default_auto_field = "django.db.models.AutoField"
```

Comment:
Suggestion: Is there a reason not to set this globally? 

`DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'` Add this to `base.py`

Reply:
Done in 81adc069